actionpack/lib/action_dispatch/routing/mapper.rb

```diff
@@ -1509,19 +1503,17 @@ def resource(*resources, &block)
         #
         #     # resource actions are at /admin/posts.
         #     resources :posts, path: "admin/posts"
-        def resources(*resources, &block)
-          options = resources.extract_options!.dup
-
-          if apply_common_behavior_for(:resources, resources, options, &block)
+        def resources(*resources, **options, &block)
```

Comment:
Likely you can extract concerns option here:

```suggestion
        def resources(*resources, concerns: nil, **options, &block)
```